docs/architecture/testing/test-framework.md

```diff
@@ -4,6 +4,36 @@ This document is an overview of the Mbed TLS test framework and test tools.
 
 This document is incomplete. You can help by expanding it.
 
+## Unit tests
+
+See https://tls.mbed.org/kb/development/test_suites
+
+### Unit test descriptions
+
+Each test case has a description which succinctly describes for a human audience what the test does. The first non-comment line of each paragraph in a `.data` file is the test description. The following rules and guidelines apply:
```

Comment:
The "for a human audience" part sounds a bit strange. Are there any descriptions we use that are not intended for humans?

Reply:
It's a description for humans as opposed to a name only used by computers. But I can remove that if you find it strange.

Reply:
I'll leave the decision to a native speaker.